packages/cli/src/commands/account/list.ts

```diff
@@ -14,25 +15,21 @@ export default class AccountList extends BaseCommand {
   requireSynced = false
 
   async run() {
-    const kit = await this.getKit()
     const res = await this.parse(AccountList)
-    // Retrieve accounts from the connected Celo node.
-    const allAddresses = !res.flags.local ? await kit.connection.getAccounts() : []
-
-    // Get addresses from the local wallet.
-    const localAddresses = res.flags.local ?? true ? kit.connection.getLocalAccounts() : []
-
+    const wallet = await this.getWalletClient()
```

Comment:
one thing that would be really liked is if listing addresses did not require connecting to a node #161

Reply:
basically create a client without  hooking a chain to it